bluebottle/files/models.py

```diff
@@ -7,12 +7,6 @@
 
 class File(models.Model):
     id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
-    owner = models.ForeignKey(
```

Comment:
I added this so that we can do permission checks. But since we are not doing that now, let's leave it like this.